sphinxcontrib_images_lightbox2/lightbox2.py

```diff
@@ -35,10 +35,12 @@ def visit_image_node_html(self, writer, node):
                 ))
         writer.body.append(
             '''<img src="{src}"
+                    class="{cls}"
                     width="{width}"
                     height="{height}"
                     alt="{alt}"/>
                     '''.format(src=node['uri'],
+                               cls='align-%s' % node['align'],
```

Comment:
Yes sphinx already support those, this is how an <img> (with the Image directive) is aligned.